src/services/wrong.details.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Request } from "express";
+import { Templates } from "../types/template.paths";
+import { urlUtils } from "../utils/url";
+
+export const buildWrongDetails = (req: Request, backLink: string, returnPath: string,
+                                  stepOneHeading: string, pageHeading: string) => {
+  return {
```

Comment:
if we keep this approach then the service will need it's own unit test.

Reply:
Its is covered by the existing controller tests so anything changed here would be picked up by those tests.